source/simulator/parameters.cc

```diff
@@ -229,6 +229,11 @@ namespace aspect
                        "The name of the directory into which all output files should be "
                        "placed. This may be an absolute or a relative path.");
 
+    prm.declare_entry ("Input directory", "input",
```

Comment:
This is not a great name and I would find it better if it is inside the section of one of the plugins instead.